README.md

```
@@ -1,3 +1,98 @@
+
+- [x] 능력을 가진다.
+- [x] 서로 협력한다.
+- [x] 진영을 가진다.
+    - 초, 한으로 구성된다.
+- [x] 서로 잡을 수 있는 지 진영으로 비교한다.
+
+### 마
+
+- [x] 진영을 가진다.
+- [x] 마는 직선으로 한 칸, 대각선으로 한 칸 움직일 수 있다.
+    - [x] 아닐 경우 예외가 발생한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)